scripts/deploy_cross_chain_verification_with_requests_output_1101_zkevm.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
are *json files supposed to be in the scripts folder

Reply:
Yes, for the different deployments we decided with @AndriianChestnykh to have deployment json files in scripts that will be used to get the addresses for the different contracts that we need for `setProofRequest.ts` and `verifier-upgrade.ts` scripts.

Reply:
Finally I'll update scripts to look for this information in the `.env` file configured that we are using also for deployment